admin/broadleaf-open-admin-platform/src/main/resources/open_admin_style/js/admin/components/entityForm.js

```diff
@@ -712,6 +712,11 @@ $(document).ready(function() {
                         }
                     });
 
+                    let validationErrors = $(data).find('.fieldError');
```

Comment:
use var instead of let, we have old minification engine that complains about using let